templates/login/login.inc

```diff
@@ -12,50 +12,135 @@
  <input type="hidden" name="url" value="<?php echo htmlspecialchars((string)$vars->url) ?>" />
  <input type="hidden" name="anchor_string" id="anchor_string" value="<?php echo htmlspecialchars((string)$vars->anchor_string) ?>" />
 
-<?php foreach ($loginparams as $key => $val): ?>
-<?php if ($val['type'] == 'hidden'): ?>
-<input type="hidden" id="<?php echo $key ?>" name="<?php echo $key ?>" value="<?php echo isset($val['value']) ? htmlspecialchars((string)$val['value']) : '' ?>" />
```

Comment:
Where did handling of hidden form params go? These are necessary for some login drivers.